doc/dialects/db2.md

```diff
@@ -56,7 +56,7 @@ The SQL statement below creates the adapter script, defines the Java class that
 ```sql
 CREATE OR REPLACE JAVA ADAPTER SCRIPT ADAPTER.JDBC_ADAPTER AS
   %scriptclass com.exasol.adapter.RequestDispatcher;
-  %jar /buckets/<BFS service>/<bucket>/virtual-schema-dist-8.0.0-bundle-4.0.5.jar;
+  %jar /buckets/<BFS service>/<bucket>/virtual-schema-dist-8.0.0-bundle-5.0.0.jar;
```

Comment:
Is this really a new major version? In that case shouldn't we have more major versions?

Reply:
well, as we change the interfaces (in this case we remove a whole bunch of it), I would say yes.

And second question, probably yes. @redcatbear what do you think?

Reply:
If and only if you make the interface incompatible, increase the major version.